src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -93,10 +93,26 @@ public T capture() {
      * 
      * @return captured argument value
      */
-    public T getValue() {
+    public T getLastValue() {
```

Comment:
Yes that's what I also thought. Make it deprecated and internally call the new one (which also has the same javadoc)